app/domain/operations/hbx_enrollments/drop_enrollment_members.rb

```diff
@@ -52,10 +55,45 @@ def drop_enrollment_members(params)
         Success(dropped_member_info)
       end
 
+      def remove_dropped_enr_members(new_enrollment, non_eligible_members)
+        ineligible_applicant_ids = non_eligible_members.pluck(:applicant_id)
+        new_enrollment.hbx_enrollment_members.where(:applicant_id.in => ineligible_applicant_ids).delete
+      end
+
+      def terminate_base_enrollment(base_enrollment, new_enrollment, terminate_date)
+        return if  base_enrollment.coverage_expired?
```

Comment:
this line not needed, current functionality does not support dropping dependent from expired enrollment.

Reply:
reinstatement had different use case that why we had expired check their, this not needed for dep drop tool.

Reply:
Removing this line.